app/web/redirects.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   redirects: [
     {
-      source: "/contributors/form",
+      source: "/volunteer/form",
```

Comment:
wrong url?

Reply:
The form should be associated with the portal from now